valen-export-gltf/src/main/java/be/twofold/valen/export/gltf/mappers/GltfModelMapper.java

```diff
@@ -43,13 +43,15 @@ Optional<MeshPrimitiveSchema> mapMeshPrimitive(Mesh mesh) throws IOException {
 
         // Have to fix up the joints and weights first
         fixJointsAndWeights(mesh);
-
         var attributes = new HashMap<String, AccessorID>();
         for (var vertexBuffer : mesh.vertexBuffers()) {
             var semantic = vertexBuffer.info().semantic();
             if (semantic == Semantic.COLOR) {
                 // TODO: Make Blender ignore vertex colors
-                continue;
+                // continue;
+                var semanticString = "_" + (vertexBuffer.info().customName() != null ? vertexBuffer.info().customName() : mapSemantic(semantic) + numColors++);
```

Comment:
Is this the only place where the `customName` property is used? And why is it specific to `COLOR` only? You could've provided a custom semantic `Custom(String name)` instead of adding a new whole property that is used _sometimes_. Doesn't seem like the best solution, but it is definitely better than the current approach.